CHANGES.md

```diff
@@ -1,5 +1,7 @@
 # git master
 
+* [540](https://github.com/Eyescale/CMake/pull/540):
+  Add CommonSmokeTest.cmake
```

Comment:
Elaborate a little, like:
Add CommonSmokeTest.cmake to check correct execution of installed applications